treesit-auto.el

```diff
@@ -138,7 +139,8 @@ automatic installation (or prompting, based on the value of
       :lang 'elixir
       :ts-mode 'elixir-ts-mode
       :remap 'elixir-mode
-      :url "https://github.com/elixir-lang/tree-sitter-elixir")
+      :url "https://github.com/elixir-lang/tree-sitter-elixir"
+      :requires '(heex))
```

Comment:
Question: is the idea here that in order to run `elixir-ts-mode`, we need both the elixir tree-sitter grammar *and* the heex one?  I would think that it goes the other way around, or that they are essentially independent, like how the TypeScript/TSX ones operate.  So, would we want a `:requires 'elixir` in the HEEX recipe, too?

I am unfamiliar with how the elixir/heex editing landscape operates, so I might just need some clarity on the desired user experience when using `treesit-auto` here.

Reply:
elixir-ts-mode requires heex-ts-mode as heex templates can be embedded into .ex files as package requires:

```
;; Package-Requires : ((emacs "29") (heex-ts-mode "1.0"))
```

Reply:
Got it; in which case it makes perfect sense for opening an elixir file to auto-install both the elixir grammar and the heex grammar